tests/e2e/features/createSpace.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Create a space
+    As a super user
+    I want to create a space
+    So that I can add recipes and share my recipes
+
+    Background:
+        Given the user has signed up with the following details
+            | username         | superuser |
+            | password         | super1234 |
+        And the user has logged in with username "superuser" and password "super1234"
+
+
+    Scenario: user creates a space
+        When the user creates a space "user's Space"
+        And the user should get success message "You have successfully created your own recipe space. Start by adding some recipes or invite other people to join you."
+        And user space "user's Space" should be visible on option menu
+
+
+    Scenario: user creates a space from space overview page
+        Given the user has browsed to the space overview page
+        When the user creates a space "user's new Space"
+        And the user should get success message "You have successfully created your own recipe space. Start by adding some recipes or invite other people to join you."
+        And user space "user's new Space" should be visible on option menu
+
+
+    Scenario: user tries to create space with existing name
+        Given the user has browsed to the space overview page
+        And the user has created a space "user's new Space"
+        And the user has browsed to the space overview page
+        When the user tries to create a space "user's new Space"
+        Then the user should get error message "Name already taken."
```

Comment:
```suggestion
Feature: Create a space
  As a super user
  I want to create a space
  So that I can add recipes and share my recipes

  Background:
    Given the user has signed up with the following details
      | username | superuser |
      | password | super1234 |
    And the user has logged in with username "superuser" and password "super1234"


  Scenario: user creates a space
    When the user creates a space "user's Space"
    And the user should get success message "You have successfully created your own recipe space. Start by adding some recipes or invite other people to join you."
    And user space "user's Space" should be visible on option menu


  Scenario: user creates a space from space overview page
    Given the user has browsed to the space overview page
    When the user creates a space "user's new Space"
    And the user should get success message "You have successfully created your own recipe space. Start by adding some recipes or invite other people to join you."
    And user space "user's new Space" should be visible on option menu


  Scenario: user tries to create space with existing name
    Given the user has browsed to the space overview page
    And the user has created a space "user's new Space"
    And the user has browsed to the space overview page
    When the user tries to create a space "user's new Space"
    Then the user should get error message "Name already taken."
```

Reply:
2 spaces as tab size are not mandatory. And the tab size is purely one's choice. @KarunAtreya and @nirajacharya2 can decide what they want to have in this project. But needs to be consistent in all the feature files

Reply:
I know it looks weird to me. I just mentioning that